src/import_fieldinformation.py

```diff
@@ -46,122 +47,108 @@ def main(path):
+                model_field = properties['model_field']
     
-    rows = 0
-    while curr_row < num_rows:
-        curr_row += 1
-        actual_row = curr_row + 1
-        row = worksheet.row(curr_row)
-        if(logger.isEnabledFor(logging.DEBUG)): logger.debug(str(('row', row)))
-        i = -1
-        initializer = {}
-        while i < num_cells:
-            i += 1
-            # Cell Types: 0=Empty, 1=Text, 2=Number, 3=Date, 4=Boolean, 5=Error, 6=Blank
-            #cell_type = worksheet.cell_type(curr_row, curr_cell)
-            value = unicode(worksheet.cell_value(curr_row, i))
-
-            if i not in cols: 
-                continue
-            properties = cols[i]
-
-            logger.debug(str(('read col: ', i, ', ', properties)))
-            required = properties['required']
-            default = properties['default']
-            converter = properties['converter']
-            model_field = properties['model_field']
-
-            # Todo, refactor to a method
-            logger.debug(str(('raw value', value)))
-            if(converter != None):
-                logger.debug(str(('using converter',converter,value)))
-                value = converter(value)
-                logger.debug(str(('converted',value)))
-            if(value == None ):
-                if( default != None ):
-                    value = default
-            if(value == None and  required == True):
-                raise Exception('Field is required: %s, record: %d' % (properties['column_label'],actual_row))
-            logger.debug(str(('model_field: ' , model_field, ', value: ', value)))
-            initializer[model_field] = value
-
-        try:
-            logger.debug(str(('initializer: ', initializer)))
-            #if((initializer['table'] == None and initializer['queryset'] == None ) or
-            if(initializer['field'] == None):
-                logger.warn(str(('Note: table entry has no field definition (will be skipped)', initializer, 'current row:', actual_row)))
-                continue;
-            lfi = FieldInformation(**initializer)
-            # check if the table/field exists
-            if(lfi.table != None):
-                table = models.get_model(APPNAME, lfi.table)
-                if( table != None):
-                    if(lfi.field not in map(lambda x: x.name,table._meta.fields) ):
-                        raise Exception(str(('unknown field: ', lfi.field)))
-                else:
-                    raise Exception(str(('unknown table', lfi.table )))
-            lfi.save()
-            logger.info(str(('fieldInformation created:', lfi)))
-            rows += 1
-        except Exception, e:
-            logger.error(str(( "Invalid fieldInformation, initializer so far: ", initializer, 'current row:', actual_row,e)))
-            raise e
+                # Todo, refactor to a method
+                logger.debug(str(('raw value', value)))
+                if(converter != None):
```

Comment:
Sorry, should have read ahead some more... also use `is` when comparing to `True` and `False`.